src/follower.py

```diff
@@ -37,17 +38,16 @@ def __init__(self):
         self._camera = Camera()
         self.reset_camera()
 
-        self._tag = TagRecognition()
+        self._tag = TagRecognition(marker_length=0.025)
```

Comment:
This is already in the repo. It's from my last commit. Remove this change and the repo again.